### DISCUSSION TITLE
自定义顶栏 频道入口归类与显示问题

### DISCUSSION THREAD

**Discussion by pinaZC** [the1812/Bilibili-Evolved#4243] [功能建议 / Ideas]
https://github.com/the1812/Bilibili-Evolved/discussions/3258#discussion-4016272
自定义顶栏都加入创作中心了，为什么不能把频道也直接加到这个功能里，非要做成个插件呢？显示上对不齐是最难受的
（谢谢大佬优化这个只有我用的b站功能）
![image](https://github.com/the1812/Bilibili-Evolved/assets/72682556/2e5db221-6ed3-4868-8b90-d44c78623e1f)

**Comment by Geeyun-JY3** [comment]
破站频道功能已被砍：#4614。本 issue 已过时，可关闭。
相关：#4623
